spec/install/post_bundle_message_spec.rb

```diff
@@ -81,6 +81,22 @@
         expect(out).to include(bundle_complete_message)
       end
     end
+
+    describe "with misspelled or non-existent gem name" do
```

Comment:
Bear with me, but I don't know what you mean by "one full test."  Does that mean you'd prefer the test style in the Resolver spec of the "post bundle message" spec?